data/hai/hai ships.txt

```diff
@@ -1075,6 +1075,114 @@ ship "Shield Beetle" "Shield Beetle (Tracker)"
 
 
 
+# Flagship of the Unfettered Hai, filled with the spoils of wars against Korath and Wanderers:
+ship "Emperor Beetle"
+	sprite "ship/hai emperor beetle"
+	thumbnail "thumbnail/hai emperor beetle"
+	attributes
+		category "Heavy Warship"
+		cost 35100000
+		shields 26700
+		hull 19600
+		"required crew" 83
+		bunks 91
+		mass 870
+		drag 9.8
+		"outfit space" 887
+		"cargo space" 27
+		"weapon capacity" 364
+		"engine capacity" 150
+		"heat dissipation" 0.675
+		# Expensive features inherited from Ladybug-era technology, with some fuel (and improved heat dissipation):
+		"tactical scan power" 30
+		ramscoop 1.5
+		"fuel capacity" 800
```

Comment:
How expensive are these really? How much outfit space would using outfits for this stuff cost? How many credits would those outfits cost?

Reply:
300 outfit cap and the ramscoop pretty much is just a fuel processor-ish
The scanner is 4 outfit space
So totalling idk... 20 outfit space and not so many creds
Its expensive because its using the ladybug tech to do it

Reply:
It's expensive because it's bad, then? In that case, it should be valued more highly than 20 tonnes of outfit space and that should be taken into account when considering how much actual outfit space the ship should have.

Reply:
Its expensive because it doesnt cost outfit space because its the hull being changed Ladybug kind (meant for deep space exploration)

Reply:
Though looking at it it should probably be tuned down to 0.5, smth similar to what the ladybug has (0.25)?

Reply:
Changed it and removed the huge added fuel, in favor of a fuel processor and a wand ramscoop (overkill but hey I switched out the medium syscore for a small one and didnt want to put outfit space down for sure, I would like to up it but I need ur opinion on it)